assets/js/modules/adsense/components/setup/SetupAccountCreate.js

```diff
@@ -31,19 +31,26 @@ import Link from '../../../../components/Link';
 import { trackEvent } from '../../../../util';
 import { parseAccountID } from '../../util/parsing';
 import { STORE_NAME } from '../../datastore/constants';
-import { STORE_NAME as siteStoreName } from '../../../../googlesitekit/datastore/site/constants';
-import { STORE_NAME as userStoreName } from '../../../../googlesitekit/datastore/user/constants';
+import { STORE_NAME as CORE_SITE } from '../../../../googlesitekit/datastore/site/constants';
+import { STORE_NAME as CORE_USER } from '../../../../googlesitekit/datastore/user/constants';
 import {
 	ErrorNotices,
 	UserProfile,
 } from '../common';
 const { useSelect } = Data;
 
 export default function SetupAccountCreate() {
-	const siteURL = useSelect( ( select ) => select( siteStoreName ).getReferenceSiteURL() );
-	const userEmail = useSelect( ( select ) => select( userStoreName ).getEmail() );
+	const siteURL = useSelect( ( select ) => select( CORE_SITE ).getReferenceSiteURL() );
+	const userEmail = useSelect( ( select ) => select( CORE_USER ).getEmail() );
 	const existingTag = useSelect( ( select ) => select( STORE_NAME ).getExistingTag() );
 	const signUpURL = useSelect( ( select ) => select( STORE_NAME ).getServiceCreateAccountURL() );
+	const supportURL = useSelect(
+		( select ) => select( CORE_SITE ).getGoogleSupportURL(
+			{
+				path: 'https://support.google.com/adsense/answer/2659101',
+			}
+		)
+	);
```

Comment:
@asvinb let's use the compact version for this hook. The same applies to other occurrences as well.

```suggestion
	const supportURL = useSelect( ( select ) => select( CORE_SITE ).getGoogleSupportURL( {
		path: 'https://support.google.com/adsense/answer/2659101',
	} ) );
```

Reply:
Updated @eugene-manuilov